radis/test/io/test_hdf5.py

```diff
@@ -64,6 +65,19 @@ def test_hdf5_io_engines(*args, **kwargs):
     df.close()
     assert manager.read_metadata("test_h5py.h5", key=None) == metadata0
 
+    # Test get_columns function of DatabaseManager
+    # For vaex
+    dbmanager = DatabaseManager(
+        name="test", local_databases="hitran", molecule="CO", engine="vaex"
+    )
+    assert list(dbmanager.get_columns("test_h5py.h5")) == ["a", "b"]
```

Comment:
Nice :+1:

Reply:
Actually @Supriya1702, can't you move get_columns() in DataFileManager?

Reply:
actually @Supriya1702 last change; since the get_columns() is in DataFileManager can you call it directly from `manager` in the example above ? 

Also, make sure that you also write a test for the engine="pytables" version

Reply:
I've done the changes